.travis.yml

```diff
@@ -7,7 +7,7 @@ rvm:
 gemfile:
   - gemfiles/ar_4.2.gemfile
   - gemfiles/ar_5.2.gemfile
-  - gemfiles/ar_6.0.gemfile
+  - gemfiles/ar_6.1.gemfile
```

Comment:
Just curious why `gemfiles/ar_6.0.gemfile` is removed?

Reply:
Oh, I didn't notice, but I should have supported both versions Rails 6.0 and 6.1...